modules/swagger-models/src/main/java/com/wordnik/swagger/models/parameters/AbstractSerializableParameter.java

```diff
@@ -0,0 +1,153 @@
+package com.wordnik.swagger.models.parameters;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.wordnik.swagger.models.properties.Property;
+
+import java.util.List;
+
+@JsonPropertyOrder({"name", "in", "description", "required", "type", "items", "collectionFormat", "default", "maximum", "exclusiveMaximum", "minimum", "exclusiveMinimum"})
+public abstract class AbstractSerializableParameter<T extends AbstractSerializableParameter> extends AbstractParameter implements SerializableParameter {
+  protected String type;
+  protected String format;
+  protected String collectionFormat;
+  protected Property items;
+  protected List<String> _enum;
+  protected Boolean exclusiveMaximum;
+  protected Double maximum;
+  protected Boolean exclusiveMinimum;
+  protected Double minimum;
+
+  @JsonProperty("default")
+  protected String defaultValue;
+
+  public T property(Property property) {
+    this.setProperty(property);
+    return (T) this;
+  }
+
+  public T parameterType(String type) {
```

Comment:
This will break the signature for the model.  While it may be more clear, I think we are too far along in the release of 1.5 to do this.